.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.8.1
+      - image: circleci/node:10.17.0
```

Comment:
should this be 10.16.2?

Reply:
i emulated the version we use in `mfp-core`. so if it should be 10.16.2 it should probably be changed a lot more places than here...